gcn/notices/fermi/gbm/alert.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "GCN Schema for Fermi GBM Alert Notices",
+  "description": "This schema is used to report Fermi GBM Alerts.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "../../core/AdditionalInfo.schema.json"
+    },
+    {
+      "$ref": "../../core/Alert.schema.json"
+    },
+    {
+      "$ref": "../../core/DateTime.schema.json"
+    },
+    {
+      "$ref": "../../core/Event.schema.json"
+    },
+    {
+      "$ref": "../../core/GeoLocBase.schema.json"
+    },
+    {
+      "$ref": "../../core/Reporter.schema.json"
+    },
+    {
+      "$ref": "../../core/Statistics.schema.json"
+    },
+    {
+      "$ref": "detectors.schema.json"
+    }
+  ],
+  "properties": {
+    "notice_type": {
+      "type": "string",
+      "description": "Notice title"
+    },
+    "human_in_the_loop": {
+      "type": "boolean",
+      "description": "Someimes abbreviated as \"HITL\".  When false, the notice was generated entirely automatically.  A value of true means that a human had an input, while the notice was being prepared.  The latter includes notices, which weren't altered by the human input, but a human was given the option to make an input, or asked to confirm, or approve before further processing."
+    },
+    "trigger_algorithm_number": {
+      "type": "number",
+      "description": "The number identifying the algorithm that detected the gamma-ray burst and caused the trigger.  \"Trigger algorithm\" here is a moniker for a method or a criterion that watches for significant rate increase in the observed rate.  Multiple \"trigger algorithms\" are active simultaneously and any one can trigger independently.  The trigger_algorithm_number, tells which one did."
```

Comment:
Are these trigger algorithms described somewhere in a document or paper. Can you just reference that document with something like the following.  There's too much extra information in this description.
"The instrument flight software algorithm that triggered on this event, dependent on timescale, energy, and significance as described in ...".